### PR TITLE
Added Home Page

### PR DESCRIPTION
**Pull request by Shah-Arsalan**
### Note:
Check the index.html and Index.css file for home page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (thegeorgenikhil) [inline index.html:141]: prettify index.html

**Review by thegeorgenikhil — CHANGES_REQUESTED** [review]

**Comment by Rohanmond** [comment]
looking so cool..just do the indentation

**Review by Rohanmond — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (Shah-Arsalan) [reply index.html:141]: Done , can you approve now?

**Review by samyakshah3008 — APPROVED** [review]
Good job, Arsalan 
Looks good to merge!

**Pull request merged**